server/src/streaming/session.rs

```diff
@@ -2,23 +2,23 @@ use std::fmt::Display;
 
 // This might be extended with more fields in the future e.g. custom name, permissions etc.
 #[derive(Debug)]
-pub struct UserContext {
+pub struct Session {
     pub user_id: u32,
```

Comment:
for future, consider having something like
`type UserId: u32`

so it won't get messed up so easily when calling functions whose belong to UserContext